cmsplugin_articles_ai/factories.py

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8 -*-
+import random
+import string
+from datetime import timedelta
+
+import factory
```

Comment:
But even having factory-boy based factories adds in several heavy packages (and also increases the chance of version conflicts).

To be honest, these factories could be easily implemented as regular functions...